actors/market/tests/deal_termination.rs

```diff
@@ -0,0 +1,221 @@
+use fil_actor_market::{DealSettlementSummary, EX_DEAL_EXPIRED};
+use fil_actors_runtime::EPOCHS_IN_DAY;
+use fvm_shared::{clock::ChainEpoch, econ::TokenAmount, error::ExitCode};
+
+mod harness;
+use harness::*;
+use num_traits::Zero;
+
+#[test]
+fn deal_is_terminated() {
+    struct Case {
+        name: &'static str,
+        deal_start: ChainEpoch,
+        deal_end: ChainEpoch,
+        activation_epoch: ChainEpoch,
+        termination_epoch: ChainEpoch,
+        settlement_epoch: ChainEpoch,
+        payment: TokenAmount,
```

Comment:
Rename to `termination_payment`, since otherwise it looks like an expected settlement payment.
Consider re-ordering the fields so settlement_epoch is last, too